manifest.yml

```diff
@@ -0,0 +1,8 @@
+---
+applications:
+- name: htbhf-java-common
+  buildpacks:
+  - java_buildpack
+  env:
+    JBP_CONFIG_OPEN_JDK_JRE: '{jre: { version: 11.+ }}'
+    APP_VERSION: ((app-version))
```

Comment:
Could remove `APP_VERSION` - and maybe whole `manifest.yml`

Reply:
Manifest isn't need if we're not deploying to cloud foundry.